src/ResourceManager/SiteRecovery/Commands.SiteRecovery/help/Edit-AzureRmSiteRecoveryRecoveryPlan.md

```diff
@@ -128,7 +130,7 @@ Accept wildcard characters: False
 ```
 
 ### -AddProtectedEntities
-Specifies an array of protected entities to add.
+An instance of an ASRProtectionEntity array of protected entities to add.
```

Comment:
An instance of an array seems confusing. This is an array of ASRProtectionEntity instances to add